okhttp/src/jvmTest/kotlin/okhttp3/internal/http/HttpUpgradesTest.kt

```diff
@@ -82,12 +82,12 @@ class HttpUpgradesTest {
         socket.sink.buffer().use { sink ->
           socket.source.buffer().use { source ->
             sink.writeUtf8("client says hello\n")
-            sink.flush()
+            sink.emit()
```

Comment:
Is this an optimization?

Reply:
Kinda sort of. I wanted to confirm that we weren’t unnecessarily wrapping the streams in unnecessary buffering.